db/migrate/20151110150858_add_content_id_to_redirect.rb

```diff
@@ -0,0 +1,12 @@
+class AddContentIdToRedirect < ActiveRecord::Migration
+  def up
+    add_column :newest_redirects, :content_id, :string
+    execute 'UPDATE newest_redirects SET content_id = UUID()'
+    change_column :newest_redirects, :content_id, :string, null: false
```

Comment:
You can use `change_column_null` for this to provide a clearer indication of intent: http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_column_null